packages/core/src/connectivity/scp-cf/connectivity-service.spec.ts

```diff
@@ -1,17 +1,17 @@
 import { createLogger } from '@sap-cloud-sdk/util';
-import { Destination } from '../../src';
-import { addProxyConfiguration } from '../../src/connectivity/scp-cf';
 import {
   mockConnectivityServiceBinding,
   mockedConnectivityServiceProxyConfig,
   mockServiceBindings
-} from '../test-util/environment-mocks';
+} from '../../../test/test-util/environment-mocks';
```

Comment:
```suggestion
} from '@sap-cloud-sdk/test-util';
```

Reply:
Also applies to all the imports below

Reply:
@florian-richter But this is a different test-util.

Reply:
To make it clear: this is `test-util` directory under the `core` package instead of the `test-util` package.